src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -622,25 +622,29 @@ std::vector<Feature> Renderer::Impl::queryRenderedFeatures(const ScreenLineStrin
         }
     }
 
+    return queryRenderedFeatures(geometry, options, layers);
+}
+    
+std::vector<Feature> Renderer::Impl::queryRenderedFeatures(const ScreenLineString& geometry, const RenderedQueryOptions& options, const std::vector<const RenderLayer*>& layers) const {
     std::unordered_set<std::string> sourceIDs;
     for (const RenderLayer* layer : layers) {
         sourceIDs.emplace(layer->baseImpl->source);
     }
-
+    
```

Comment:
nit: Tabs on empty lines

Reply:
`Renderer::queryPointAnnotations` calls `queryRenderedFeatures` specifying the layer it should query. In this case point annotations are rendered in a single layer. You can pass all the layer ID's for the shape annotations; it was already originally designed this way as you can read in this [comment](https://github.com/mapbox/mapbox-gl-native/pull/9984#discussion_r140621967) we decided to change that to let the rendered figures out all the shape layer IDs on its own. 

Which name do you think it would be more explicit than `queryShapeAnnotations`?

Reply:
I think `queryRenderedSourceFeatures` could be renamed to `queryShapeAnnotationFeatures` or better yet collapsed into `queryShapeAnnotations`